.github/workflows/quick-test.yml

```diff
@@ -45,27 +45,33 @@ jobs:
         os: ['windows-2016', 'windows-latest']
         include:
           - os: windows-2016
-            target: 'Ninja'
+            target: 'Visual Studio 15 2017'
             arch:
```

Comment:
Missing arch `-A x64`

Reply:
I think this was intentional. I tried it both ways. See my earlier comments in this thread. I was hoping to do a 32-bit build in order to get 32-bit Windows coverage without having to further expand the build matrix. But, I couldn't figure out how to get OpenSSL. With the 64-bit build, meanwhile, I was encountering random segfaults (which don't happen under VS 2019).